client/Windows/wf_floatbar.c

```diff
@@ -25,6 +25,7 @@
 #include "wf_client.h"
 #include "wf_floatbar.h"
 #include "wf_gdi.h"
+#pragma comment( lib, "Msimg32.lib" )
```

Comment:
What's the minimum OS version Msimg32.lib is available? - Wasn't that windows 2000?